spec/src/test/java/impl/UserResourceTest.java

```diff
@@ -178,11 +270,15 @@ public void shouldReturnInternalServerErrorIfUserInAuthCannotBeFound() {
     }
 
     private User insertUser() {
+        return insertUser("Test Subject", "picture");
+    }
+
+    private User insertUser(String name, String picture) {
         final String generatedEmail = UUID.randomUUID().toString() + "@fortnox.se";
```

Comment:
I realize it's outside of your changed scope, but wouldn't it be better to use example.com here instead of a real domain?

Reply:
Yes thats true!